### DISCUSSION TITLE
Accessibility: Shortcuts for View Tasks and Switching Tasks

### DISCUSSION THREAD

**Discussion by marcusquinn** [minbrowser/min#1876] [Completed]
I really like the concept of one window browsing, and a new window closes all the existing tabs into past Tasks.

However, I mostly navigate anything using the keyboard, and really miss shortcuts for switching Tasks (Windows).

I see that Ctrl-Tab switches Tabs forwards, and Ctrl-Shift-Tab backwards, and that makes sense.

How about Ctrl-Alt-Tab & Ctrl-Alt-Shift-Tab for switching Tasks (Windows) please?

View Tasks doesn't have any shortcut or link under the text menus. How about Ctrl-Alt-W to toggle that view please?

**Comment by PalmerAL** [comment]
There's some more shortcuts in preferences > keyboard shortcuts that aren't listed in the main menu:

* ctrl + [ and ctrl + ] switch between tasks
* shift+ctrl+e shows the task view

(These are editable also)

Ctrl+alt+tab seems like a logical enough shortcut, maybe we should add that by default as well.

**Comment by marcusquinn** [comment]
I see, I missed that, although I think I'll change to command-shift-w (Mac).

I think it would help to add Menu options under the View menu for:
* Previous Task `Ctrl/Cmd [`
* Next Task `Ctrl/Cmd ]`
* Show All Tasks `Ctrl Shift E`

PS, the Tasks feature makes this my favourite browser now!
